introduction.md

```diff
@@ -1,2 +1,8 @@
 Hello, I am working on "Mastering GitHub". I am a developer in test for USA TODAY/Gannett.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?